lib/cocina/models/catalog_link.rb

```diff
@@ -10,7 +10,7 @@ class CatalogLink < Struct
       attribute :refresh, Types::Strict::Bool.default(false)
       # Record identifier that is unique within the context of the linked record's catalog.
       # example: 11403803
-      attribute :catalogRecordId, Types::Strict::String
+      attribute :catalogRecordId, Types::Strict::String.constrained(format: /^\d+(:\d+)*$/)
```

Comment:
Here you can see the regex lifted out of OpenAPI-land into Rubyland.

Reply:
nice.